about/src/main/java/dev/arkbuilders/components/about/ContextUtils.kt

```diff
@@ -0,0 +1,35 @@
+package dev.arkbuilders.components.about
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.widget.Toast
+
+internal fun Context.openLink(url: String, showToastIfFailed: Boolean = true) {
```

Comment:
These two methods share the same exeption handling code, I think it's good that we can use HoC to avoid duplicate code. Something like:

```kotlin
private fun wrappedLaunch(action: () -> Unit) {
  try {
    action()
  } catch(e: Throwable) {
    if (showToastIfFailed) {
      Toast.makeText(
          this,
          getString(R.string.oops_something_went_wrong),
          Toast.LENGTH_SHORT
      ).show()
  }
  }
}

internal fun Context.openEmail(mailTo: String, showToastIfFailed: Boolean = true) {
    wrappedLaunch {
        val uri = Uri.parse("mailto:$mailTo")
        startActivity(Intent(Intent.ACTION_SENDTO, uri))
    }
}

internal fun Context.openLink(url: String, showToastIfFailed: Boolean = true) {
    wrappedLaunch {
       startActivity(Intent(Intent.ACTION_VIEW).setData(Uri.parse(url)))
    }
}
```